frontend/src/components/FAQ/CollapsibleHeader.tsx

```diff
@@ -35,7 +32,7 @@ export default function CollapsibleHeader({ headerName, faqs }: Props): ReactEle
 
         <Collapse {...collapseProps}>
           <CardBody className={styles.sectionBody}>
-            {faqs? faqs.map((faq, index) => (<CollapsibleQuestion key={index} {...faq} />)) : null}
+            {faqs ? faqs.map((faq, index) => (<CollapsibleQuestion key={index} {...faq} />)) : null}
```

Comment:
If there's nothing to render if `faqs` is falsy, you can just condense this line to  `{faqs && faqs.map((faq, index) => (<CollapsibleQuestion key={index} {...faq} />))}`